### PR TITLE
fix: PayloadTooLargeError: request entity too large

### PR DESCRIPTION
**Pull request by tonybentley**
Resolves issue #732

Bumps the file upload limit size to 10 mb

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -48,10 +48,14 @@ const express = require('express'),
   compareVersions = require('compare-versions')
 
 function Server (opts) {
+  const bodyParser = require('body-parser')
   const app = express()
   app.use(require('compression')())
   app.use(require('cors')())
-  app.use(require('body-parser').json())
+  app.use(bodyParser.json({limit:'50mb'}))
+  app.use(bodyParser.urlencoded({
```
Comment (tkurki) [inline lib/index.js]: Why is urlencoded needed to solve the size limit?

File: lib/index.js [hunk 2]
```diff
@@ -48,10 +48,14 @@ const express = require('express'),
   compareVersions = require('compare-versions')
 
 function Server (opts) {
+  const bodyParser = require('body-parser')
   const app = express()
   app.use(require('compression')())
   app.use(require('cors')())
-  app.use(require('body-parser').json())
+  app.use(bodyParser.json({limit:'50mb'}))
```
Comment (tkurki) [inline lib/index.js]: 50mb is a pretty hefty global default.

How about making this configurable with an environment variable and making the default something more reasonable, like 10mb?

File: lib/index.js (hunk 1 above)
Comment (tonybentley) [reply lib/index.js]: Updated after testing. Not needed as you suspected.

File: lib/index.js (hunk 2 above)
Comment (tonybentley) [reply lib/index.js]: Added environment variable and default to 10mb.

File: lib/index.js [hunk 3]
```diff
@@ -48,10 +48,12 @@ const express = require('express'),
   compareVersions = require('compare-versions')
 
 function Server (opts) {
+  const FILEUPLOADSIZELIMIT = process.env.FILEUPLOADSIZELIMIT || '10mb'
```
Comment (tkurki) [inline lib/index.js:51]: Could you please add a note about this to README.md?

Comment (tonybentley) [reply lib/index.js:51]: done

**Comment by tkurki** [comment]
Thank you very much! I recall there have been earlier issues about this, glad to have it fixed.

**Pull request merged**